src/terminal/parser/ut_parser/InputEngineTest.cpp

```diff
@@ -352,6 +352,9 @@ void InputEngineTest::C0Test()
         case L'\t': // Tab
             writeCtrl = false;
             break;
+        case L'\b': // backspace
+            wch = '\x7f';
+            expectedWch = '\x7f';

```

Comment:
nit: this should probably have a break too, even though it's the last case

Reply:
Makes sense, I included it in the latest commit

Reply:
Makes sense, the latest commit added the break statement